Assets/Scripts/Utilities/SceneManagerProjectPorcupine.cs

```diff
@@ -0,0 +1,80 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using UnityEngine.SceneManagement;
+
+public class SceneManagerProjectPorcupine
```

Comment:
I also think that it is more of a Controller and should go in the Controllers folder.